src/gui/sdlmain.cpp

```diff
@@ -2225,8 +2225,8 @@ void GFX_SetMouseRawInput(const bool requested_raw_input)
 	if (SDL_SetHintWithPriority(SDL_HINT_MOUSE_RELATIVE_MODE_WARP,
 	                            requested_raw_input ? "0" : "1",
 	                            SDL_HINT_OVERRIDE) != SDL_TRUE)
-		LOG_WARNING("SDL: Mouse raw input %s failed",
-		            requested_raw_input ? "enable" : "disable");
+		LOG_WARNING("SDL: Failed %s raw mouse input",
+		            requested_raw_input ? "enabling" : "disabling");
```

Comment:
"Failed to enable/disable raw mouse input" - I think reads better here.

Reply:
Changed.